konfera/tests/test_views.py

```diff
@@ -81,6 +92,16 @@ def test_cfp_non_existing_event(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, 404)
 
+    def test_cfp_not_allowed(self):
+        url = reverse('event_cfp_form', kwargs={'slug': 'cfp-not-allowed'})
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 404)
+
+    def test_cfp_after_deadline(self):
+        url = reverse('event_cfp_form', kwargs={'slug': 'passed-cfp'})
+        response = self.client.get(url)
+        self.assertIn('{% trans "Thank you for your interest, but unfortunately', str(response.content))
```

Comment:
It is not necessary to test whether the translation was applied or not. 

But you could do it with https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#explicitly-setting-the-active-language . You activate a different language for example `sk` and then check if the page shows the text in Slovak.